src/app/components/select-language/select-language.component.html

```diff
@@ -1,13 +1,20 @@
-<ion-select
-  *ngrxLet="currentLanguageData$ as currentLanguageData"
-  class="tw-w-fit"
-  interface="action-sheet"
-  [value]="currentLanguageData.code"
-  [cancelText]="'button.cancel' | translate"
-  (ionChange)="handleLanguageChange($event)">
-  <span class="fi fi-{{ currentLanguageData.flagIconCode }}" slot="label"></span>
+<ion-item class="tw-relative" *ngrxLet="currentLanguageData$ as currentLanguageData">
```

Comment:
this component also used in public header, so it is better to wrap in ion-item in spcific place it is needed only